### DISCUSSION TITLE
Promote some (latency-improving) Experimental features to non-experimental

### DISCUSSION THREAD

**Discussion by PallHaraldsson** [JuliaLang/julia#43707] [General]
I know some TTFP issues solved successfully with the mechanism used in e.g. Plots.jl and many other packages, including some where I've proposed it, such as in this declined PR: https://github.com/gcalderone/Gnuplot.jl/pull/27#issuecomment-1004715134

There's some resistance to using those features, for 2 reasons. a) as in there, since it's "Experimental", which might be a good reason, if the features where ever going away (or are buggy); and b) in case slowing down non-startup code (not claimed there, nor I think a real worry).

I believe I have good overview of the latency-improving solutions; these two in Base, the original one used as above in PR, and the later more general one (enabling e.g. "--compile=min"). Some questions, are there more, in Base? And could at least those two be made non-experimental? I don't think they're ever going away, as they do work, but in theory such could, and/or the API be changed, to no longer indicate (mislead) as experimental; I suppose if the API changes then the old one is also kept? 

In general is there some process or policy to know about how such matters are handled, or other stuff that should be reconsidered similarily?

**Comment by vtjnash** [comment]
I converted this question to a discussion, though it might be more appropriate for discourse.
